drivers/SmartThings/matter-button/src/init.lua

```diff
@@ -69,6 +69,7 @@ end
 
 --end of helper functions
 --------------------------------------------------------------------------
+-- TODO: remove this and use device:find_default_endpoint once those lua libs changes are widely available
```

Comment:
nit: I think you should remove this comment, because Im not sure this function will ever be in the lua libs.

Reply:
I agree with your feedback and have closed the associated lua libs PR and removed the comments here and in the other drivers 👍